<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Histogram</class>
 <widget class="QDialog" name="Histogram">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>402</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="0,0,0">
    <item row="1" column="0">
     <widget class="QGraphicsView" name="graphicsView_Red"/>
    </item>
    <item row="1" column="1">
     <widget class="QGraphicsView" name="graphicsView_Green"/>
    </item>
    <item row="1" column="2">
     <widget class="QGraphicsView" name="graphicsView_Blue"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labelRed">
      <property name="text">
       <string>Red</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="labelGreen">
      <property name="text">
       <string>Green</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="labelBlue">
      <property name="text">
       <string>Blue</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
